sormas-backend/src/main/java/de/symeda/sormas/backend/importexport/ImportFacadeEjb.java

```diff
@@ -193,6 +189,8 @@
 	private RegionService regionService;
 	@EJB
 	private CountryFacadeEjbLocal countryFacade;
+	@EJB
+	private ImportParserService converterService;
```

Comment:
Shouldn't this variable be called importParserService instead of converterService?